tests/test_2886.py

```diff
@@ -0,0 +1,27 @@
+import fitz
+
+import os
+
+
+def test_2886():
+    """Confirm correct insertion of a 'named' link."""
+    path = os.path.abspath(f"{__file__}/../../tests/resources/cython.pdf")
```

Comment:
Suggest disabling this test when run with classic with:
```
def test_2886():
    if not hasattr(fitz, 'mupdf'):
        print(f'test_2886(): not running on classic.')
        return
    ...
```

Reply:
great - will do